src/components/Property.astro

```diff
@@ -18,11 +18,14 @@ const link = customDataTypes.includes(formattedType) ? '/api/data-types#' + form
     </dd>
     <dt class="sr-only">Type</dt>
     <dd>
-      <Badge text={formattedType} link={link}>
+      <Badge text={ error ? type : formattedType} link={link}>
```

Comment:
Suggestion: It might be nicer to have the conditional logic at the top.
```
const formattedType = error ? type.toUpperCase : type.toLowerCase();
```

Reply:
@alexBinns @AndyClifford actually maybe we should have a separate component for errors. The fields aren't really description and type, they are status code and error message.

Reply:
Yea I started doing that but thought what's the point of a whole new component for different variable names

Reply:
@alexBinns  If you read the code or read this on a screen reader it makes no sense with the existing titles 🤔

Reply:
I'll approve to unblock but I do think it would be tidier to have separate fields 🙌